scss/detail.scss

```diff
@@ -1,18 +1,42 @@
 .book-container {
-  border: 1px solid $light-gray;
+  background-color: $white;
+  box-shadow: 6px 6px 5px rgba($black, 0.4);
+  display: flex;
+  left: 50%;
+  margin: 20px;
+  padding: 25px;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 800px;
+}
+
+.book-cover {
+  height: 340px;
+  width: 240px;
+}
+
+.book-information {
   display: flex;
   flex-direction: column;
-  height: 195px;
+  height: 220px;
   justify-content: space-between;
```

Comment:
it should not has `height` fixed. Try removing that `height` and you find other _value_ of `justify-property`

Reply:
ready